src/components/CreateTarget/CreateTarget.js

```diff
@@ -42,9 +49,18 @@ const CreateTarget = () => {
       return;
     }
 
-    const target = buildTargetRequest();
-    const created = await TargetsService.create(target);
-    if (created) {
+    const targetData = buildTargetRequest();
+    const { target, matched_user, match_conversation } =
```

Comment:
use camel case style
```suggestion
    const { target, matched_user: matchedUser, match_conversation } =
```